src/dependencies/xerces-c/CMakeLists.txt

```diff
@@ -32,10 +32,10 @@ if (WIN32)
     set( _BUILD_DIR Build/Win32/VC${MSVC_VERSION}/${_DEBUG_OR_RELEASE} )
   endif()
 
-  if (MSVC_VERSION GREATER 12)
-    # Only have project files for VS12 and older
-    set(MSVC_VERSION 12)
-  endiF()
+  #if (MSVC_VERSION GREATER 12)
```

Comment:
xerces-c does not have project files for VC>14. You should limit the MSVC_VERSION locally in the module to 14 otherwise xerces-c will not compile. See the `projects\Win32\` directory in the xerces-c distribution for list of supported VC.